core/src/commonMain/kotlin/com/ustadmobile/core/controller/EpubContentPresenter.kt

```diff
@@ -131,6 +132,8 @@ class EpubContentPresenter(context: Any,
 
     var mCurrentPage: Int = 0
 
+    var contextRegistration: String? = null
```

Comment:
At the moment we are only using the context registration at the end, in onStop. Let's just have it as a local variable.

For variables that are setup (and never null) in onCreate and then used later on in the lifecycle, we can use lateinit. We know that onCreate will always be called first. We don't need nullability.